src/shop-client.js

```diff
@@ -142,7 +142,7 @@ const ShopClient = CoreObject.extend({
    * ```
    *
    * @method fetchQuery
-   * @public
+   * @private
```

Comment:
looking at the docs it does say "methods that _shouldn't_ be used externally" so I guess it works either way.